sdk/core/dma/dma.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
```

Comment:
This probably doesn’t belong in the core directory, that’s for core parts of the RTOS, whereas this is optional.